### ISSUE TITLE
Deployment 

### ISSUE THREAD

**Issue by Tvaahr** [nbgallery/nbgallery#927]
At my company we use NBgallery it is a great way for our scientist to share notebooks. Thank you for the great work!

I have a question regarding your deployment procedure. Is it intentional that you push a new build to both the latest and rail6 docker tags simultaneously?   

The reason I ask is that we apply some of our own patches, configs, and extensions by having our own DockerFile, which builds on top of your docker image. When we are upgrading our docker image, or developing on the extensions around it, we would like the starting image to be static to not have changed sneak in during our development. 

The behavior I would expect is that the latest tag would be the very latest build of the application and rail6 (o other tag) would be static.

Thanks in advance.

**Comment by mcrutch** [comment]
The rails6 tag was created when we were in the migration from rails4->rails 5->rails 6 and just tracks with latest since latest is rails 6.  I never got around to making versioned releases for the NBG container, mostly because no one asked more than anything.  It wouldn't be too hard to start doing tagged releases and have CI/CD publish images from those tags, but we will probably do it in bunches of merge requests rather than every merge to main.

**Comment by Tvaahr** [comment]
Lumping merge request in release tags makes sense to me and would be greatly appreciated, while still having the latest tag up to date with the main branch.

**Comment by mcrutch** [comment]
Started fresh at 1.0.0 (published now) going forward will tag periodically.

**Comment by Tvaahr** [comment]
Thanks a lot. I test it with my docker build

**Comment by Tvaahr** [comment]
When I build on the docker image tagged 1.0.0 the build succeeds, but I get this error starting NBgallery from the browser. Building from the latest tag works fine. 

```bash
D, [2023-09-21T07:35:32.107930 #46] DEBUG -- :   UserPreference Load (0.5ms)  SELECT `user_preferences`.* FROM `user_preferences` WHERE `user_preferences`.`user_id` = 577 LIMIT 1                                                                                                                                           I, [2023-09-21T07:35:32.108575 #46]  INFO -- :   Rendered static_pages/home.slim within layouts/layout (Duration: 2.2ms | Allocations: 457)                                                                                                                                                                                  I, [2023-09-21T07:35:32.108665 #46]  INFO -- :   Rendered layout layouts/layout.slim (Duration: 2.4ms | Allocations: 501)                                                                                                                                                                                                    I, [2023-09-21T07:35:32.109005 #46]  INFO -- : Completed 500 Internal Server Error in 32ms (ActiveRecord: 10.5ms | Allocations: 6363)                                                                                                                                                                                        F, [2023-09-21T07:35:32.109892 #46] FATAL -- :
ActionView::Template::Error (The asset "nbgallery_logo_white.png" is not present in the asset pipeline.
):
    23:       -user_pref = UserPreference.find_by(user_id: @user.id)
    24:       div id="hiddenSpinner" role="alert"
    25:         -if user_pref != nil && (user_pref.theme == "dark" || user_pref.theme == "ultra-dark")
    26:           ==image_tag("nbgallery_logo_white.png", class:"center ld ld-breath logo-loading", alt:"Content is loading")
    27:         -else
    28:           ==image_tag("nbgallery_logo.png", class:"center ld ld-breath logo-loading", alt:"Content is loading")
    29:       nav.main.content id="txt" aria-label="home notebook tabs"

app/views/static_pages/home.slim:26
I, [2023-09-21T07:35:32.110478 #46]  INFO -- : Processing by Gaffe::ErrorsController#show as HTML
```

**Comment by mcrutch** [comment]
I can't duplicate that.  I did the following
```git pull
git checkout 1.0.0
docker build .

**Comment by mcrutch** [comment]
Published 1.0.0 image worked fine for me as well.  Not sure what could have caused that problem.

**Comment by Tvaahr** [comment]
Thanks for investigating. It must be something on my side I will see if I can figure out what is going on
